float32 distance_meters
string zone  # "CLEAR", "CAUTION", "DANGER"
int32 left_pwm
int32 right_pwm
float32 left_steering
float32 right_steering